config/app.js

```diff
@@ -14,27 +18,27 @@ export default {
    *
    * @type {Boolean}
    */
-  debug: process.env.NODE_ENV !== 'production',
+  debug: env('NODE_ENV') !== 'production',
```

Comment:
Would we want to default to `production` here? To ensure if somehow omitted it doesn't reveal debug info on a prod env? I think Aaron made a change like this in Phoenix which seemed like a good idea.

Reply:
Oh, I like that idea! 👍

Reply:
Added this in a5f1e81! It'll now throw an exception if `NODE_ENV` is invalid.